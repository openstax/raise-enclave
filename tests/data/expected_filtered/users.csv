uuid,first_name,last_name,email
d35a986b-a7ad-4c61-8e76-205eccaf9a06,Chris,Pirelli,chrispirelli13431@gmail.us