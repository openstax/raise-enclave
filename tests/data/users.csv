sourcedId,enabledUser,orgSourcedIds,role,username,userIds,givenName,familyName,middleName,identifier,email,sms,phone,agentSourcedIds,grades,password
U100,,,,,,Margarite,Bell,,,margueritebell53245@gmail.com,,,,,
U101,,,,,,Deann,James,,,deannjames121231@gmail.com,,,,,
U102,,,,,,James,Stewart,,,jamesstewart1234121@gmail.com,,,,,
U103,,,,,,David,Tucci,,,davidtucci13431@gmail.us,,,,,