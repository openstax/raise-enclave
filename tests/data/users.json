[
  {
    "department": "",
    "description": "",
    "descriptionformat": 1,
    "email": "davidtucci13431@gmail.us",
    "enrolledcourses": [
      {
        "fullname": "Algebra 1 (Research Dev)",
        "id": 2,
        "shortname": "Alg1"
      }
    ],
    "firstaccess": 1661273594,
    "firstname": "David",
    "fullname": "David Tucci",
    "groups": [],
    "id": 7,
    "uuid": "66730745-7ee0-444b-904e-c137c5334aab",
    "lastaccess": 1661273865,
    "lastcourseaccess": 1661273865,
    "lastname": "Tucci",
    "profileimageurl": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f1",
    "profileimageurlsmall": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f2",
    "roles": [
      {
        "name": "",
        "roleid": 5,
        "shortname": "student",
        "sortorder": 0
      }
    ],
    "username": "davidtucci13431@gmail.us"
  },
  {
    "department": "",
    "description": "",
    "descriptionformat": 1,
    "email": "deannjames121231@gmail.com",
    "enrolledcourses": [
      {
        "fullname": "Algebra 1 (Research Dev)",
        "id": 2,
        "shortname": "Alg1"
      }
    ],
    "firstaccess": 1661273125,
    "firstname": "Deann",
    "fullname": "Deann James",
    "groups": [],
    "id": 11,
    "uuid": "546f4dbc-9488-4147-87d4-c2bae1bd8b97",
    "lastaccess": 1661273186,
    "lastcourseaccess": 1661273202,
    "lastname": "James",
    "profileimageurl": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f1",
    "profileimageurlsmall": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f2",
    "roles": [
      {
        "name": "",
        "roleid": 5,
        "shortname": "student",
        "sortorder": 0
      }
    ],
    "username": "deannjames121231@gmail.com"
  },
  {
    "department": "",
    "description": "",
    "descriptionformat": 1,
    "email": "doncastle1234123@gmail.com",
    "enrolledcourses": [
      {
        "fullname": "Algebra 1 (Research Dev)",
        "id": 2,
        "shortname": "Alg1"
      }
    ],
    "firstaccess": 1661272974,
    "firstname": "Don",
    "fullname": "Don Castle",
    "groups": [],
    "id": 10,
    "uuid": null,
    "lastaccess": 1661273103,
    "lastcourseaccess": 1661273105,
    "lastname": "Castle",
    "profileimageurl": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f1",
    "profileimageurlsmall": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f2",
    "roles": [
      {
        "name": "",
        "roleid": 4,
        "shortname": "teacher",
        "sortorder": 0
      }
    ],
    "username": "doncastle1234123@gmail.com"
  },
  {
    "department": "",
    "description": "",
    "descriptionformat": 1,
    "email": "jamesstewart1234121@gmail.com",
    "enrolledcourses": [
      {
        "fullname": "Algebra 1 (Research Dev)",
        "id": 2,
        "shortname": "Alg1"
      }
    ],
    "firstaccess": 1661273287,
    "firstname": "James",
    "fullname": "James Stewart",
    "groups": [],
    "id": 8,
    "uuid": "757555fb-3044-4164-99a9-5705d3bf4d5a",
    "lastaccess": 1661278589,
    "lastcourseaccess": 1661278591,
    "lastname": "Stewart",
    "profileimageurl": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f1",
    "profileimageurlsmall": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f2",
    "roles": [
      {
        "name": "",
        "roleid": 5,
        "shortname": "student",
        "sortorder": 0
      }
    ],
    "username": "jamesstewart1234121@gmail.com"
  },
  {
    "department": "",
    "description": "",
    "descriptionformat": 1,
    "email": "margueritebell53245@gmail.com",
    "enrolledcourses": [
      {
        "fullname": "Algebra 1 (Research Dev)",
        "id": 2,
        "shortname": "Alg1"
      }
    ],
    "firstaccess": 1661272048,
    "firstname": "Marguerite",
    "fullname": "Marguerite Bell",
    "groups": [],
    "id": 9,
    "uuid": "c6a4a01e-b112-475f-82ff-b051ab1d44e1",
    "lastaccess": 1661272665,
    "lastcourseaccess": 1661272665,
    "lastname": "Bell",
    "profileimageurl": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f1",
    "profileimageurlsmall": "https://research-dev.raiselearning.org/theme/image.php/boost/core/1660688913/u/f2",
    "roles": [
      {
        "name": "",
        "roleid": 5,
        "shortname": "student",
        "sortorder": 0
      }
    ],
    "username": "margueritebell53245@gmail.com"
  }
]