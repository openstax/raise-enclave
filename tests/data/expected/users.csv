uuid,first_name,last_name,email
66730745-7ee0-444b-904e-c137c5334aab,David,Tucci,davidtucci13431@gmail.us
546f4dbc-9488-4147-87d4-c2bae1bd8b97,Deann,James,deannjames121231@gmail.com
7080c78d-298b-40ba-a68d-55d6a93b00fb,Don,Castle,doncastle1234123@gmail.com
757555fb-3044-4164-99a9-5705d3bf4d5a,James,Stewart,jamesstewart1234121@gmail.com
c6a4a01e-b112-475f-82ff-b051ab1d44e1,Marguerite,Bell,margueritebell53245@gmail.com
d35a986b-a7ad-4c61-8e76-205eccaf9a06,Chris,Pirelli,chrispirelli13431@gmail.us